drafts/draft-20005.adoc

```diff
@@ -0,0 +1,225 @@
+
+[#OLL]
+=== Open Liberty console logging now has the ability to format logs with date and time stamps and other relevant information
+
+In Liberty, users have the option to format their server logs that appear in the console.log file in different formats, such as JSON and dev (default) formats, in their server logging configuration, using the consoleFormat logging attribute. The dev format will show messages in a basic format, where there are no timestamp or any other relevant information. It will only show the message with its log level and the message itself.
+
+For example:
+[source, xml]
+----
+consoleFormat=dev (default)
+[AUDIT ] CWWKE0001I: The server server1 has been launched.
+----
+
+This feature introduces a new option called `simple` for the consoleFormat logging server configuration attribute. This new option will configure Liberty to output logs in the same simple (default) format as the message.log file, with date/time stamps and other relevant information, to the `console.log` file or to the console (Standard-out).
+
+For example:
+
+[source, xml]
+----
+consoleFormat=simple
+[25/11/19 10:02:30:080 EST] 00000001 com.ibm.ws.kernel.launch.internal.FrameworkManager A CWWKE0001I: The server server1 has been launched.
+----
+
+To configure the Liberty logs to output logs in the new simple console format, you just have to set the following logging server configuration, in the bootstrap.properties, server.xml, or in the server.env (environment variables).
+
+==== server.env
+`WLP_LOGGING_CONSOLE_FORMAT=simple`
+
+==== bootstrap.properties
+`com.ibm.ws.logging.console.format=simple`
+
+==== server.xml
+`<logging consoleFormat="simple"/>`
+
+
+
+[#JLO]
+=== Omit specified fields from JSON logging output
+
+In Liberty, users have the option to format their server logs in a json format. When logs are in JSON format, users have to specify the sources (message, trace, accessLog, ffdc, audit) they want to send to messages.log or console.log/standard-out.
+
+Users can now specify the JSON fields they want to omit. This feature adds an option for users to omit JSON fields in the JSON logging process. The option to omit JSON field names in Liberty is extremely useful, as users may not want certain default fields provided by Liberty in their JSON output. Undesired fields add to the size of the records which wastes network I/O during record transmissions and waste space in downstream log aggregation tools. Now users have the option to only emit the fields they need so they can send to downstream log aggregation tools without using more space and I/O than necessary.
+For someone running liberty in docker containers, with a single server in each container, they may not want to include the JSON fields that represent the server name and user directory, for example.
```

Comment:
>The option to omit JSON field names in Liberty is extremely useful, as users may not want certain default fields provided by Liberty in their JSON output. Undesired fields add to the size of the records which wastes network I/O during record transmissions and waste space in downstream log aggregation tools. Now users have the option to only emit the fields they need so they can send to downstream log aggregation tools without using more space and I/O than necessary.
For someone running liberty in docker containers, with a single server in each container, they may not want to include the JSON fields that represent the server name and user directory, for example.

suggested update:

>The option to omit JSON field names in Open Liberty is extremely useful, as users might not want certain default fields provided by Open Liberty in their JSON output. Undesired fields add to the size of the records, which wastes network I/O during record transmissions and wastes space in downstream log aggregation tools. Now, users can choose to emit only the fields they need so they can send to downstream log aggregation tools without using more space and I/O than necessary. For example, someone who's running Open Liberty in Docker containers, with a single server in each container, might not want to include the JSON fields that represent the server name and user directory.